bin/cron_crab_unique_users.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+OUTPUT_DIR="${1:-./output}"
```

Comment:
Could you please help section such that if we run script w/o arguments or with `-h`, `--help` we can get usage message.